### ISSUE TITLE
File Context Menu "Open Archive" does not open any archive or other files...

### ISSUE THREAD

**Issue by YuryKomar** [ip7z/7zip#38]
When select file and right click on it, then select a File Context Menu "Open Archive" - it does not open selected file as archive.

Any format does not react to the "Open Archive" context menu item...

How to fix it?

My OS: Windows 10 x64

Earlier verions, like 9+ worked before just fine.

**Comment by ip7z** [comment]
Try another commands in menu.
Try to run `7zFM.exe` and `7zG.exe` from `C:\Program Files\7-Zip\` frolder.

**Comment by YuryKomar** [comment]
> Try another commands in menu. Try to run `7zFM.exe` and `7zG.exe` from `C:\Program Files\7-Zip\` folder.

Thanks for reply.

I have just fixed the issue.
It was long time ago my mistake, I have set to the file '7zFM.exe' (in a file properties windows) a tick to checkbox "Run As Administrator", to be able to run the app always as admin in order to apply all settings without always restart as admin.

So, I have now UNTICK that checkbox (in Explorer file Properties Window) and running the menu "Open Archive" works fine...

But anyway, all other menu items were run as usual, only Open Archive was the problem.

I think the case closed.
